backends/ebpf/ebpfTable.h

```diff
@@ -72,7 +72,11 @@ class EBPFTable : public EBPFTableBase {
  protected:
     const cstring prefixFieldName = "prefixlen";
 
-    bool isLPMTable();
+    bool isLPMTable() const;
+    bool isTernaryTable() const;
+
+    void emitTernaryInstance(CodeBuilder* builder);
```

Comment:
Maybe `emitTableInstance`? It is not cleat why this name.

Reply:
keep in mind that this is a method of "table" object. So, we shouldn't have a call to table.emitTableInstance()